packages/apps/hr/src/screens/leaveRequests/LeaveListScreen.tsx

```diff
@@ -55,6 +60,34 @@ const LeaveListScreen = ({}) => {
     [dispatch, selectedStatus, user?.id],
   );
 
+  const sendLeaveAPI = useCallback(
+    item => {
+      dispatch(
+        (sendLeave as any)({
+          leaveRequestId: item.id,
+          version: item.version,
+          userId: user?.id,
+          selectedStatus,
```

Comment:
Create a useMemo with these data (leaveRequestId, version and selectedStatus) and use it with ... to avoid repetition